ansible/roles/providers/tasks/main.yml

```diff
@@ -88,6 +88,15 @@
     mode: 0644
   when: ansible_os_family == "Debian"
 
+- name: set cloudinit feature flags for Red Hat
+  copy:
+    src: usr/lib/python3/dist-packages/cloudinit/feature_overrides.py
+    dest: "{{ sysusr_prefix }}/lib/python3.6/site-packages/cloudinit/feature_overrides.py"
+    owner: root
+    group: root
+    mode: 0644
+  when: ansible_os_family == "RedHat"
```

Comment:
We can just add this to the above `when` condition:

```
when: ansible_os_family in ["Debian", "RedHat"]
```

Reply:
Notice the very slight difference between the paths dist vs site